app/javascript/application.js

```diff
@@ -0,0 +1,18 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+// import "./controllers"
```

Comment:
Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :pray: